front-end/src/components/Content/User/style.scss

```diff
@@ -0,0 +1,67 @@
+$backgrount-color: white;
+$border-color: #ccc;

```

Comment:
```SCSS
$backgrount-color: white;
$border-color: #ccc;
```
You could more to _variable.scss, because in the UI we can re-use anywhere and change the color anywhere quickly at one file.
Thanks